examples/angular/scripts/build.js

```diff
@@ -1,31 +1,45 @@
 const { spawn, spawnSync } = require("node:child_process");
+const path = require("node:path");
 const ls = spawn("node", ["-v"]);
 
-ls.stdout.on("data", (data) => {
+ls.stdout.on("data", data => {
 	let res = data.toString().trim();
 	const [a, _] = res.split(".");
 	if (a >= "v16") {
 		build()
 			.then(console.log)
-			.catch((err) => {
+			.catch(err => {
 				console.error(err);
-				process.exit(-1);
+				process.exit(1);
 			});
 	} else {
-		console.log("The Angular CLI requires a minimum of v16.13")
+		console.log("The Angular CLI requires a minimum of v16.13");
 		// Working around angular not support node14, but we need to test node v14 in CI
 		process.exit(0);
 	}
 });
 
-ls.stderr.on("data", (data) => {
+ls.stderr.on("data", data => {
 	throw new Error(`${data}`);
 });
 
 function build() {
 	return new Promise((resolve, reject) => {
 		try {
-			spawnSync("npx", ["rspack", "build"], { stdio: "inherit" });
+			const { status, error, stderr } = spawnSync(
+				"node",
+				[
+					path.resolve(__dirname, "../node_modules/@rspack/cli/bin/rspack"),
```

Comment:
Didn't know why this binary is not linked to the `.bin` directory in windows, and I used this as a workaround.

Reply:
Changes in this PR should not affect this. Does the CI run failed?

Reply:
Yes. `npx` is not working as expected since `bin` was not exist there.

Reply:
> Yes. npx is not working as expected since bin was not exist there.

Does the main branch have the problem too?

Reply:
> > Yes. npx is not working as expected since bin was not exist there.
> 
> Does the main branch have the problem too?

I don't think this will execute the correct one on the fly as `npx` may also pull the package from remote. This might be the reason that `main` is working. Using this technique is to make sure to test locally linked packages.